schemas/oic.sec.sp.name.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.sec.sp.name.json#",
+  "title": "Security Profile Name",
+  "definitions": {
+    "oic.sec.sp.name": {
+          "type": "string",
+          "enum": ["oic.sec.sp.unspecified", "oic.sec.sp.baseline", "oic.sec.sp.black", "oic.sec.sp.blue"],
+          "description": "A string specifying the security profile",
+          "detail-desc": ["oic.sec.sp.unspecified - The security profile is not specified",
```

Comment:
not sure if detail-desc is a tag one can use.

Reply:
detail-desc is used in many Security schemas, where enums are used. So added here in same way to be consistent with the others.
We can discuss, if we need to move detail-desc text into description.

Reply:
i can't find anything back in json schema that this is a valid tag.

Reply:
Need to add oic.sec.sp.purple to this enum.